admin/src/components/Tables/OpenCreationsTable.vue

```diff
@@ -168,14 +174,26 @@ export default {
       required: false,
     },
   },
-  emits: ['update-contributions', 'reload-contribution', 'update-status', 'show-overlay'],
+  emits: [
+    'update-contributions',
+    'reload-contribution',
+    'update-status',
+    'show-overlay',
+    'search-for-email',
+  ],
```

Comment:
the same here for predefined emit-strings and line 219?

Reply:
https://vuejs.org/guide/components/events.html